frontend/style.css

```diff
@@ -269,11 +269,17 @@
 .drink_element {
   padding: 20px;
   width: 350px;
-  height: 600px;
+  height: 400px;
```

Comment:
It is a bad idea to use `px` in styling, as it is not responsive. Imagine using this app on your phone, on a small screen or on an extra large screen. Read more about using `em`, `rem` and other values.
https://www.w3.org/Style/Examples/007/units.pl.html

Reply:
ok, I will use em or rem or other better values